### ISSUE TITLE
AResult:Error: Error executing ctags

### ISSUE THREAD

**Issue by draeder** [ennucore/clippy#3]
I'm getting the following error:

```
AResult:Error: Error executing ctags: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ctags: illegal option -- -
usage: ctags [-BFadtuwvx] [-f tagsfile] file ...
```

Is this due to a typo in the Poetry install [documentation](https://python-poetry.org/docs/#installing-with-the-official-installer)? Specifically, the dash at the end: `curl -sSL https://install.python-poetry.org | python3 -`

**Comment by ennucore** [comment]
The issue is with ctags. Did you [install](https://docs.ctags.io/en/latest/building.html) the correct ctags from README?

**Comment by draeder** [comment]
Yes, I followed the instructions from [here](https://docs.ctags.io/en/latest/osx.html) since I am using MacOS.

**Comment by tfedoseev** [comment]
Did you install it using [brew](https://brew.sh)? It looks like the version of ctags within Xcode is being called.

**Comment by draeder** [comment]
Yes, I followed the [instructions](https://docs.ctags.io/en/latest/osx.html#building-with-homebrew) which use brew.
